c_src/bloom_filter.hpp

```diff
@@ -268,6 +268,7 @@ class bloom_filter
          {
             bit_table_[i] |= filter.bit_table_[i];
          }
+         inserted_element_count_ += filter.inserted_element_count_;
```

Comment:
Technically not right, because if both contain a given value, it'll be double-counted. But doing it right would be O(n). (A logical "and" could find all the dupes, but then you'd still need to count them up to subtract from the total, and that would still be O(n))

Reply:
Yeah, true that, I naively implemented this and forgot to follow up on the issue when it popped in my brain 😮‍💨